frontend/src/pages/about/About.module.css

```
@@ -0,0 +1,58 @@
+.container {
+  background: #1a1a1a;
+  color: white;
+  font-family: 'Inter', sans-serif;
```

Stylelint (17.2.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)